Cargo.lock

```diff
@@ -2059,6 +2059,7 @@ dependencies = [
  "serde",
  "serde_json",
  "serde_urlencoded",
+ "system-configuration",
```

Comment:
Hmmm I'm surprised there's a new dependency. Might be interesting to see what they added it for or if it's a feature we don't need and can turn off in our `Cargo.toml` for `reqwest`

Reply:
Right. I tried cherry-picking features for our dependencies in the `Cargo.toml`. The build times were benefited. Everything seemed to work fine.